.gitignore

```diff
@@ -17,3 +17,7 @@ test_report.xml
 /build/
 /dist/
 /docs/api/
+
+# Ignore user-defined chip samples, but allow example chip sample
+src/qiskit_iqm/fake_backends/chip_samples/*
+!src/qiskit_iqm/fake_backends/chip_samples/example_sample.py
```

Comment:
This is unnecessary. Typical use case of  `qiskit-iqm` does not involve working with git. As for developers, let's leave it as their responsibility to take care of files they created for whatever reason